AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -397,7 +397,7 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
 
         DispatchQueue.global().async {
             self.audioLock.lock()
-            self.audioClient.stopSession()
+            _ = self.audioClient.stopSession()
```

Comment:
What causes these changes?

Reply:
updated the description. Output is not being used which causes issue, so explicitly saying we don't use output.